pkg/errors/handler_error.go

```diff
@@ -99,6 +100,7 @@ func HttpCodeForDaoError(err error) int {
 			return http.StatusInternalServerError
 		}
 	} else {
+		zerolog.Ctx(c.Request().Context()).Err(err).Msg("Internal Server Error Generated")
```

Comment:
is this still needed?

Reply:
nope!  removed and reverted that part of the pr